hasher-matcher-actioner/scripts/hma_shell

```diff
@@ -0,0 +1,165 @@
+#! /usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Prototype of shell wrapper for HMA utils for interactive manual testings
+
+```
+python3 scripts/hma_shell --pwd <password-of-user>
```

Comment:
I tried running the shell when testing my MD5 changes. It was not clear to me which password you are talking about here.

Reply:
fair, it is meant to be the password given to a pervious call to hma_shell that used, `--create_user`